src/mir/index.md

```diff
@@ -221,26 +221,28 @@ function (along with sub-instances of Mir for "promoted constants",
 but [you can read about those below](#promoted)).
 
 - **Basic blocks**: The basic blocks are stored in the field
-  `basic_blocks`; this is a vector of `BasicBlockData`
-  structures. Nobody ever references a basic block directly: instead,
-  we pass around `BasicBlock` values, which are
-  [newtype'd] indices into this vector.
-- **Statements** are represented by the type `Statement`.
+  [`basic_blocks`][basicblocks] of the struct [`Body`][Body]; this is a vector 
+  of [`BasicBlockData`][BasicBlockData] structures. Nobody ever references a
+  basic block directly: instead, we pass around [`BasicBlock`][BasicBlock]
+  values, which are [newtype'd] indices into this vector.
+- **Statements** are represented by the type [`Statement`][Statement].
 - **Terminators** are represented by the `Terminator`.
-- **Locals** are represented by a [newtype'd] index type `Local`. The
-  data for a local variable is found in the `Mir` (the `local_decls`
+- **Locals** are represented by a [newtype'd] index type [`Local`][Local]. 
+  The data for a local variable is found in the `Mir` (the `local_decls`
```

Comment:
Interestingly, this is outdated, `Mir` was renamed to `Body`... two years ago??? 😅

Nothing to do with this PR though

Reply:
@LeSeulArtichaut I'm happy to send a second PR cleaning it up :)